src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/dashboard/dashboard.component.html

```diff
@@ -1,32 +1,57 @@
-<main aria-label="Dashboard" >
+<main aria-label="Dashboard">
   <a href="#main"
      class="sr-only">skip to content</a>
 
-  <cd-refresh-selector></cd-refresh-selector>
+  <div class="container-fluid">
+    <div class="row d-flex justify-content-end">
+      <div *ngIf="dashboardVersion === 'v2'"
+           class="col-sm-1 col-wider">
+        <cd-refresh-selector></cd-refresh-selector>
+      </div>
+      <div class="col-sm-1">
+        <div class="form-check form-switch">
+          <input class="form-check-input"
+                 type="checkbox"
+                 [checked]="dashboardVersion === 'v3'"
+                 (change)="toggleContent()">
+          <label class="form-check-label"
+                 for="flexSwitchCheckDefault">
+            Dashboard V3
```

Comment:
And don't forget to i18n.
```suggestion
            Dashboard v3
```